geonode/geoserver/geofence.py

```diff
@@ -203,38 +207,64 @@ def add_insert_rule(self, rule: Rule):
         operation.update(rule.get_object())
         self.operations.append(operation)
 
-    def get_batch_length(self):
+    def length(self) -> int:
```

Comment:
Isn't this a property more than a method?

Reply:
By leaving it as a method, it can be extended by filtering only certain types of operations